github/provider.go

```diff
@@ -374,3 +385,30 @@ func providerConfigure(p *schema.Provider) schema.ConfigureFunc {
 		return meta, nil
 	}
 }
+
+// See https://github.com/integrations/terraform-provider-github/issues/1822
+func tokenFromGhCli(baseURL string) (string, error) {
+	ghCliPath := os.Getenv("GH_PATH")
```

Comment:
Can you please document this new environment variable in your docs update?

Reply:
Will do